.github/workflows/tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: unit-tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -e .
+      - name: Install Chrome browser
+        run: sudo apt-get install -y chromium-browser
+      - name: Run all test cases
+        run: pytest -rA
+      - name: Run flake8
+        run: flake8 --max-line-length=120
```

Comment:
Any benefit of changing the max line length to 120

Reply:
Line length of 80 seems too restrictive. I've seen 120 followed as a standard in a few place. So, I've added it.